setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
```

Comment:
If we limit the maximum line length to 100, we should do it on all our projects.